mage_integrations/mage_integrations/connections/airtable/__init__.py

```diff
@@ -0,0 +1,31 @@
+from pyairtable import Api, Base
```

Comment:
do you need to add this package to requirements.txt?

Reply:
yes